### PR TITLE
Migrated ciChecks Module from probot to github actions

### PR DESCRIPTION
**Pull request by Ketan1502**
## Explanation

The ciChecks.js module is successfully migrated from probot to github actions as mentioned in issue #277  
## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by Ketan1502** [comment]
@vojtechjelinek please let me know if any corrections are to be made!

File: actions/src/pull_requests/ciChecks.js [hunk 1]
```diff
@@ -12,21 +12,25 @@
...
+ * @returns{Promise<Boolean>}
  */
-const handleFailure = async (context) => {
-  /**
-   * @type {import('probot').Octokit.ChecksGetSuiteResponse} checkSuite
-   */
+const handleFailure = async () => {
+  
+  const token = core.getInput('repo-token');
+  const octokit = new GitHub(token);
```
Comment (vojtechjelinek) [inline actions/src/pull_requests/ciChecks.js:33]:
```suggestion
  const octokit = new GitHub(core.getInput('repo-token'));
```

File: actions/src/pull_requests/ciChecks.js [hunk 2]
```diff
@@ -38,21 +42,18 @@ const handleFailure = async (context) => {
...
       const prAuthor = pullRequest.user.login;
 
       const commentBody =
-        'Hi @' +
-        prAuthor +
-        ', there are some failing CI checks in your latest push ' +
-        ' If you think this is due to a flake, please file an issue ' +
-        'before restarting the tests. Thanks!';
-      await pingAndAssignUsers(context, pullRequest, [prAuthor], commentBody);
+        'Hi @' + prAuthor + ', there are some failing CI checks in your latest push ' + ' If you think this is due to a flake, please file an issue ' + 'before restarting the tests. Thanks!';
```
Comment (vojtechjelinek) [inline actions/src/pull_requests/ciChecks.js:55]: Make sure to limit line lengths under 80 chars.

File: constants.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ const issuesAssignedCheck = 'issues-assigned-check';
 const forcePushCheck = 'force-push-check';
 const pullRequestReviewCheck = 'pr-review-check';
 const codeOwnerCheck = 'code-owner-check';
-const ciFailureCheck = 'ci-failure-check';
+const ciFailureCheck = 'ci-failure_check';
```
Comment (vojtechjelinek) [inline constants.js:56]: Why is this changed?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Took a pass. Why is the _.env.save_ and _package-lock.json_ file added?

File: actions/src/pull_requests/ciChecks.js (hunk 1 above)
Comment (Ketan1502) [reply actions/src/pull_requests/ciChecks.js]: done

File: actions/src/pull_requests/ciChecks.js (hunk 2 above)
Comment (Ketan1502) [reply actions/src/pull_requests/ciChecks.js]: done

File: constants.js (hunk 3 above)
Comment (Ketan1502) [reply constants.js]: sorry, corrected now!

**Comment by Ketan1502** [comment]
@vojtechjelinek any further changes required?

File: actions/src/pull_requests/ciChecks.js [hunk 4]
```diff
@@ -38,21 +41,22 @@ const handleFailure = async (context) => {
...
 
       const commentBody =
-        'Hi @' +
-        prAuthor +
-        ', there are some failing CI checks in your latest push ' +
-        ' If you think this is due to a flake, please file an issue ' +
+        'Hi @' + 
+        prAuthor + 
+        ', there are some failing CI checks in your latest push ' + 
+        ' If you think this is due to a flake, please file an issue ' +
```
Comment (vojtechjelinek) [inline actions/src/pull_requests/ciChecks.js:57]: Unneeded modifications.

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left one more comment.

**Comment by Ketan1502** [comment]
@vojtechjelinek can it be merged now?

**Comment by vojtechjelinek** [comment]
@Ketan1502 Did you test all these changes according to the instructions in the PR description.

**Comment by Ketan1502** [comment]
> @Ketan1502 Did you test all these changes according to the instructions in the PR description.

yes, all the tests have passed @vojtechjelinek

**Comment by vojtechjelinek** [comment]
@Ketan1502 But did you test your changes manually?

**Comment by Ketan1502** [comment]
I tested using npm test

**Comment by vojtechjelinek** [comment]
@Ketan1502 The tests are failing though 
image
also you need to verify that your changes are working manually see https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot and https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix

**Comment by Ketan1502** [comment]
Hey! @vojtechjelinek, I have corrected the lint checks but facing issues in the frontend checks. I have cross checked my front end checks with the frontend checks of the sample PR and both were showing the same errors but the sample PR's checks have passed but mine are not, can u please help me out to understand my fault.
Sample PR's frontend checks-
https://github.com/oppia/oppiabot/actions/runs/1071735395


MY PR's frontend checks-
https://github.com/oppia/oppiabot/actions/runs/1882562313

Capture
